views/update/update-manage-trusts-tell-us-about-the-legal-entity.html

```diff
@@ -0,0 +1,15 @@
+{% extends "update-layout.html" %}
```

Comment:
There is already a file called views/update/update-trusts-tell-us-about-the-legal-entity.html checked into main.
Same as this but no "-manage-" in name. So this is causing the code duplication analysis to fail.